us/en/skymiles/blocks/toc/toc.js

```diff
@@ -0,0 +1,16 @@
+export default function decorate(block) {
+  const headings = document.querySelectorAll('h3');
```

Comment:
Why `h3`? From a semantical point of view we have `h1` as being the page title and `h2` being all the sections beneath it.

If the goal is to avoid the 1st h2 that acts as a subtitle, then I'd suggest using something like:
```js
[...document.querySelectorAll('h2')].slice(1)
```
or
```js
[...document.querySelectorAll('h2')].filter((h2) => h2.previousElementSibling.nodeName !== 'H1')
```